esp/logging/test/logging_test.cpp

```diff
@@ -139,9 +139,9 @@ void sendRequest()
         Sleep(5000); //Waiting for loggingManager to start
         Owned<IEspLogEntry> entry = loggingManager->createLogEntry();
         entry->setOption(option.str());
-        entry->setEspContext(espContext);
-        entry->setUserContextTree(userContextTree);
-        entry->setUserRequestTree(userRequestTree);
+        entry->setEspContext(LINK(espContext));
```

Comment:
To avoid these same issues in the future, you may want to name the functions setOwnEspContext() if it takes over ownership of the interface.